### PR TITLE
Rebuild PDB documentation

### PR DESCRIPTION
**Pull request by jaredpar**
Get the rebuild PDB documentation up to date with our recent changes

### CODE REVIEW

**Comment by jaredpar** [comment]
@dotnet/roslyn-compiler @tmat  PTAL

File: docs/features/pdb-compilation-options.md [hunk 1]
```diff
@@ -257,20 +255,28 @@ Example:
...
-#### Options For CSharp
-
-See [compiler options](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/listed-alphabetically) documentation
+#### Shared Options for CSharp and Visual Basic
 
 | PDB Key                | Format                                  | Default   | Description  |
 | ---------------------- | --------------------------------------- | --------- | ------------ |
-| language               | `CSharp`                                | required  | Language name. |
-| language-version       | `[0-9]+(\.[0-9]+)?`                     | required  | [langversion](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/langversion-compiler-option) |
+| language               | `CSharp|Visual Basic`                   | required  | Language name. |
```
Comment (tmat) [inline docs/features/pdb-compilation-options.md:262]: Seems like markdown interprets `|` as table column delimiter even when enclosed in backticks :-/

Comment (tmat) [reply docs/features/pdb-compilation-options.md:262]: Maybe escaping would help `\|`?

Comment (Youssef1313) [reply docs/features/pdb-compilation-options.md:262]: I think this should fix it:

```suggestion
| language               | `CSharp\|Visual Basic`                   | required  | Language name. |
```

File: docs/features/pdb-compilation-options.md [hunk 2]
```diff
@@ -257,20 +255,28 @@ Example:
...
+
+#### Options For CSharp
+
+See [compiler options](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/listed-alphabetically) documentation
+
+| PDB Key                | Format                                  | Default   | Description  |
+| ---------------------- | --------------------------------------- | --------- | ------------ |
+| language-version       | `[0-9]+(\.[0-9]+)?`                     | required  | [langversion](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/langversion-compiler-option) |
 | define                 | `,`-separated identifier list           | empty     | [define](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/define-compiler-option) |
 | checked                | `(True|False)`                          | `False`   | [checked](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/checked-compiler-option) |
```
Comment (Youssef1313) [inline docs/features/pdb-compilation-options.md:281]:
```suggestion
| checked                | `(True\|False)`                          | `False`   | [checked](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/checked-compiler-option) |
```

File: docs/features/pdb-compilation-options.md [hunk 3]
```diff
@@ -257,20 +255,28 @@ Example:
...
 | nullable               | `(Disable|Warnings|Annotations|Enable)` | `Disable` | [nullable](https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references) |
```
Comment (Youssef1313) [inline docs/features/pdb-compilation-options.md:282]:
```suggestion
| nullable               | `(Disable\|Warnings\|Annotations\|Enable)` | `Disable` | [nullable](https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references) |
```

File: docs/features/pdb-compilation-options.md (hunk 1 above)
Comment (jaredpar) [reply docs/features/pdb-compilation-options.md]: Yeah the `\` escaping fixed it.

File: docs/features/pdb-compilation-options.md (hunk 2 above)
Comment (jaredpar) [reply docs/features/pdb-compilation-options.md]: Ha, you and I were doing this in parallel. Pushed up a new commit that should resolve all of these.

**Review by tmat — APPROVED** [review]

**Comment by tmat** [comment]
Thanks!

**Pull request merged**